src/views/RoomSelector.vue

```diff
@@ -165,9 +167,10 @@ export default {
 
 <style lang="scss" scoped>
 #modal-inner {
-	width: 90vw;
-	max-width: 400px;
-	height: 55vh;
+	width: 100%;
+	height: 100%;
+	padding: 20px;
+	margin: 0 auto;
```

Comment:
Good @marcoambrosini ?

Reply:
Hi @marcoambrosini 
Would you like to check it? :)